packages/venia-ui/lib/components/SearchBar/searchBar.css

```diff
@@ -10,6 +10,7 @@
 .root_open {
     composes: root;
     display: grid;
+    z-index: 2;
```

Comment:
kind of arbitrary, do we have anything relative to compare this to?

Reply:
I just changed it to z-index: 3 to match the account drop-down. It just needs to be more that `1` so it goes above the loading indicator which is lower down in the DOM